lib/common/network/api/characters_api.dart

```diff
@@ -46,9 +46,9 @@ class FakeCharactersApi implements CharactersApi {
   }
 
   @override
-  Future<Characters> addCharacter(@Body() Character newCharacter) async {
+  Future<void> addCharacter(@Body() Character newCharacter) async {
     await Future.delayed(const Duration(seconds: 2));
     fakeCharacters.add(newCharacter);
-    return Characters(characters: fakeCharacters);
+    print('all $fakeCharacters');
```

Comment:
remove print, we add prints only for debugging, these changes

Reply:
Done